### PR TITLE
[Half-Modular] Ashtongue for Ashwalkers

### PR DESCRIPTION
**Pull request by Nerev4r**
## About The Pull Request
Hey. It's your least favorite catgirl again. Back at it again.
image
This replaces the current Draconic that Ashwalkers speak with a new language, Ashtongue. It is not selectable by the crew, but the Curator should get it with their omnitongue. You can also attain this language through the ashen necklaces that Ashwalkers can craft, or by becoming one yourself.

## How This Contributes To The Skyrat Roleplay Experience
1) A large chunk of miners and non-miners (due to the public shuttle allowing just anyone to lavaland, which is a good thing) take draconic for the express purpose of crip walking right into the ashwalker nest to catch some indigenous poon (or other roleplay, it doesn't really matter what goes on there more as the method of acquisition matters). I equated it in dev-main ages ago to being the equivalent of 'some dude posted up in northern Florida at an outpost in the 1800s and you see some Seminole dude skulking around the Everglades and you decide 'I will now develop native-level fluency in Mikasuki all by myself so I can tap that ass.' You should have to actively pay in roleplaying with them to be able to talk to them, and if they need to talk to you *without* a necklace or anything, it's trivial to use emotes for non-verbal communication.

2) The existence of this being possible assumes that whatever language/dialect of a language ashwalkers do speak is for some reason entirely compatible with modern draconic and there is no difficulty in either party understanding each other, which is. Weird? Strange? Offputting? This also assumes Ashwalkers can perfectly grok draconic that is coming from the mouth and throat structure of someone that isn't even a lizard but we kinda lost the ability to pursue this as an issue when sylvan and xenocommon and slime became accessible to the human mouth.

3) There is something to be said about this ability wholly invalidating the existence of the ashwalker necklace and the ability to voluntarily convert someone into an ashwalker, both of which are actual cool and mystical stuff. They're far less impactful. You don't get the rp of convincing some shamanistic type into convincing them that their sacred tongue is worthy of coming out of your filthy imperialist mouth.

4) If we actually want to promote more conflict between ashies and miners (the whole curse thing in the other ritual PR was intended to do this) then we're going to have to have some level of difficulty in communicating with each other, otherwise things are going to reach a homeostasis of. well. not having any conflict at all.
So my solution, the hidden 5) section, is to give them their own language. Lorewise, it'd just be highly mutated Draconic both from long periods of developing pretty much on their own and the enigmatic influence of the Necropolis trying to pattern the way it communicates onto the mouths of mundane creatures (which wasn't entirely successful but it was something,) given these are immortal magical lizards pretty much grown out of bone and muscle and blood out of a planet-spanning entity and they're sapient from minute 1. 

## Changelog
:cl:
add: Adds a new Ashwalker-specific language, Ashtongue. Go roleplay with them if you wanna speak it too!
/:cl:

### CODE REVIEW

File: code/modules/antagonists/ashwalker/ashwalker.dm [hunk 1]
```diff
@@ -49,6 +49,17 @@
 		owner.teach_crafting_recipe(recipe_datum)
 	//SKYRAT EDIT: Recipes for Tribals
 
+//SKYRAT EDIT: Ashtongue for Ashwalkers
+/datum/antagonist/ashwalker/apply_innate_effects(mob/living/mob_override)
```
Comment (Zonespace27) [inline code/modules/antagonists/ashwalker/ashwalker.dm]: modularize this

File: code/modules/surgery/organs/tongue.dm [hunk 2]
```diff
@@ -96,7 +97,7 @@
 	say_mod = "hisses"
 	taste_sensitivity = 10 // combined nose + tongue, extra sensitive
 	modifies_speech = TRUE
-	languages_native = list(/datum/language/draconic)
+	languages_native = list(/datum/language/draconic, /datum/language/ashtongue) //SKYRAT EDIT: Ashtongue for Ashwalkers
```
Comment (Zonespace27) [inline code/modules/surgery/organs/tongue.dm:100]: doesn't this give regular lizardpeople ashtongue?

File: code/modules/language/language_holder.dm [hunk 3]
```diff
@@ -285,8 +285,11 @@ Key procs
 	spoken_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
 							/datum/language/draconic = list(LANGUAGE_ATOM))
 
-/datum/language_holder/lizard/ash
-	selected_language = /datum/language/draconic
+/datum/language_holder/lizard/ash //SKYRAT EDIT BEGIN: Ashtongue for Ashwalkers
```
Comment (Zonespace27) [inline code/modules/language/language_holder.dm:288]: leave a commented out version of the original

File: code/modules/surgery/organs/tongue.dm [hunk 4]
```diff
@@ -43,7 +43,8 @@
 		/datum/language/zolmach, // SKYRAT EDIT - customization - extra languages
 		/datum/language/xenoknockoff, // SKYRAT EDIT - customization - extra languages
 		/datum/language/yangyu, // SKYRAT EDIT - customization - extra languages
-		/datum/language/schechi // SKYRAT EDIT - customization - extra languages
+		/datum/language/schechi, // SKYRAT EDIT - customization - extra languages
+		/datum/language/ashtongue // SKYRAT EDIT - customization - extra languages
```
Comment (Zonespace27) [inline code/modules/surgery/organs/tongue.dm]:
```suggestion
		/datum/language/ashtongue, // SKYRAT EDIT - customization - extra languages
```

**Review by Zonespace27 — DISMISSED** [review]

**Comment by Zonespace27** [comment]
Would like to see a TM for this for a day+ before merge as well

File: code/modules/surgery/organs/tongue.dm (hunk 2 above)
Comment (Nerev4r) [reply code/modules/surgery/organs/tongue.dm:100]: No, this just makes it so that they don't hiss.

**Comment by Iamgoofball** [comment]
Make sure to make the lavaland draconic necklace teach you this instead of draconic, as the point of that amulet is communication with ashwalkers, since it's loot you get from doing a boss or some crap, I dunno, lavaland sucks.

**Comment by Nerev4r** [comment]
> Make sure to make the lavaland draconic necklace teach you this instead of draconic, as the point of that amulet is communication with ashwalkers, since it's loot you get from doing a boss or some crap, I dunno, lavaland sucks.

Ye, already handled.

**Comment by Pidgey-chan** [comment]
Seems an unnecessary obstacle to roleplay for this role; throwing around the magic necklace just to get a use for the T key. It's *cool* having a mutual language with regular lizards, a connection, a tiny thing held in common, but if it's not even going to be selectable then the thread is gone, and the *cool* nothing but a memory of simpler and easier times.

**Comment by GoldenAlpharex** [comment]
> Seems an unnecessary obstacle to roleplay for this role; throwing around the magic necklace just to get a use for the T key. It's _cool_ having a mutual language with regular lizards, a connection, a tiny thing held in common, but if it's not even going to be selectable then the thread is gone, and the _cool_ nothing but a memory of simpler and easier times.

The problem is that right now a very high percentage of miners have draconic as a language for this exact reason, which defeats the purpose of having a language barrier.

**Comment by Iamgoofball** [comment]
@Pidgey-chan Try finding ways to communicate non-verbally.

**Comment by Nerev4r** [comment]
josephjoestarohmygod.wav

**Comment by YakumoChen** [comment]
I think the better comparison would be Japanese and Nekomimetic. Neko is supposed to be a deviation of Japanese, but it's bastardised to the point of being completely incomprehensible to someone who knows the former.

That said I agree with the PR.

**Comment by Zonespace27** [comment]
Jake's cool with it and so am i

**Pull request merged**